kroxylicious/src/main/java/io/kroxylicious/proxy/internal/util/Metrics.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.internal.util;
+
+public class Metrics {
+
+    // creating a constant for all Metrics in the one place so we can easily see what metrics there are
+
+    public static final String kroxylicious_inbound_downstream_messages = "kroxylicious_inbound_downstream_messages";
```

Comment:
nit: by convention constant names are usually uppercase in java like `VERY_STATIC_THING`